### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Jakub-Bielawski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +26,7 @@ sudo apt install libswscale-dev libavcodec-dev libavutil-dev python3-virtualenv
 ## Cloning the repository
```
Comment (arekmula) [inline README.md:22]: Get rid of additional packages to install, as they are not needed now (they were prerequisites) used by H264Decoder.

File: README.md [hunk 2]
```diff
@@ -5,8 +5,16 @@ The drone stays in the air and watches your hand by the camera.
...
+## Steering
+- Palm -> Drawing
+- Fist -> Stop drawing
+- Two hands -> Stop Drawing
 
+![alt text](pictures/palm.png "PALM GESTURE")
+![alt text](pictures/fist.png  "FIST GESTURE")
+
+## Performance
+![Alt Text](pictures/ezgif.com-gif-maker.gif)
```
Comment (arekmula) [inline README.md]: Change filename

**Review by arekmula — APPROVED** [review]

**Review by Krisisss — APPROVED** [review]

**Review by arekmula — CHANGES_REQUESTED** [review]
Change the filename and get rid of additional packages

File: README.md (hunk 1 above)
Comment (Jakub-Bielawski) [reply README.md:22]: I'm not sure if it will work due to djitellopy package use tH264Decoder too. I'll delete and we will see :dancers:

**Review by Skwarson96 — APPROVED** [review]

**Review by arekmula — APPROVED** [review]

**Pull request merged**